omnigan/losses.py

```diff
@@ -276,6 +276,23 @@ def __call__(self, prediction, target):
         relDiff = torch.mean(d) * torch.mean(d)
         return diff - relDiff
 
+class SIGMLoss(nn.Module):
+    def __init__(self, gmweight):
```

Comment:
add `device` argument and send `self.filter` to that device in the init, not to do it at every call

Reply:
done